dashboard/migrations/0001_initial.py

```diff
@@ -154,6 +154,7 @@ class Migration(migrations.Migration):
                 ('course', models.OneToOneField(on_delete=django.db.models.deletion.CASCADE, primary_key=True, serialize=False, to='dashboard.Course', verbose_name='Course View Option Id')),
                 ('show_files_accessed', models.BooleanField(default=True, verbose_name='Show Files Accessed View')),
                 ('show_assignment_planning', models.BooleanField(default=True, verbose_name='Show Assignment Planning View')),
+                ('show_assignment_planning_v2', models.BooleanField(default=True, verbose_name='Show Assignment Planning View v2')),
```

Comment:
@justin0022 you shouldn't be editing the new migration file, not a best practice. You need to create a new one instead. I would suggest make all your model changes in `model.py` on top of master changes that way you only have one migration file-related to assignment redesign. 
`python manage.py makemigration` - this created a migration file
`python manage.py migrate`- this will migrate the changes to DataBase

Reply:
Okay good to know. 

@ssciolla and I have been working on merging the latest master into this PR, but are having some issues.